app/views/admin/announcements/_announcement.html.slim

```diff
@@ -9,7 +9,7 @@
       = edit_button([:admin, announcement])
       = delete_button([:admin, announcement])
 
-  h3.timestamp
+  div.timestamp
```

Comment:
Some were using h3 and some div, and div looked better. But yeah semantically h3 does make more sense.